src/System.Runtime/tests/System/ArrayTests.cs

```diff
@@ -2955,6 +2904,7 @@ public static IEnumerable<object[]> Sort_Array_Array_Generic_TestData()
         [Theory]
         [MemberData(nameof(Sort_Array_Array_NonGeneric_TestData))]
         [MemberData(nameof(Sort_Array_Array_Generic_TestData))]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot, "Causes an assert to be fired at runtime in uapaot")]
```

Comment:
Can you include the issue ID in your SkipOnTargetFramework text? akin to ActiveIssue. Then we can dig it up later if it never got fixed.